docs/readmes/contributing/contribute_id_mappings.md

```diff
@@ -0,0 +1,32 @@
+---
+id: contribute_id_mappings
+title: Codeowners GitHub to Slack IDs
+hide_title: true
+---
+
+Here are the Github to Slack ID mappings for all members of [magma maintainers](https://github.com/orgs/magma/teams/repo-magma-maintain)
+
+| GitHub           | Slack (email)              |Slack  (display name)            |
+|-|-|-|
+| 119Vik           | vitalii@freedomfi.com      | Vitalii Kostenko          |
```

Comment:
Side note: I use https://csvtomd.com/#/ for automatically constructing Markdown tables -- makes it super easy to get uniform formatting

Reply:
Will check that out thanks :)